platform/ios/src/MGLScaleBarView_Private.h

```diff
@@ -0,0 +1,12 @@
+#import <UIKit/UIKit.h>
+#import <CoreLocation/CoreLocation.h>
+
+#import "MGLScaleBarView.h"
+
+@interface MGLScaleBarView (Private)
```

Comment:
I like @friedbunny's suggestion of exposing some style properties. Perhaps alternating colors, border color etc and then we'll need this private header.